test/test_po_files.py

```diff
@@ -50,6 +50,7 @@ class TestLocale(unittest.TestCase):
         "zh_TW",  # Chinese Traditional
         "ko_KR"]  # korean
 
+    @unittest.skip("Test requires unavailable locales.")
```

Comment:
i'd rather remove this test entirely, like it was done in newer branches already